packages/h2-async/h2-async.0.9.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "Antonio Monteiro <anmonteiro@gmail.com>"
+authors: [ "Antonio Monteiro <anmonteiro@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anmonteiro/ocaml-h2"
+bug-reports: "https://github.com/anmonteiro/ocaml-h2/issues"
+dev-repo: "git+https://github.com/anmonteiro/ocaml-h2.git"
+doc: "https://anmonteiro.github.io/ocaml-h2/"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {>= "1.7"}
+  "faraday-async" {>= "0.8.2"}
+  "async"
```

Comment:
Optional since it's already in as part of gluten-async but i suppose it's a bit better to be more explicit
```suggestion
  "async" {>= "v0.15.0"}
```